src/components/pages/Offer/OfferDetails/OfferCover/OfferCoverPlaceholder/OfferCoverPlaceholder.jsx

```diff
@@ -2,11 +2,11 @@ import React from 'react'
 
 import { ReactComponent as AddMediation } from './assets/add-mediation.svg'
 
-const OfferPreviewPlaceholder = () => (
+const OfferCoverPlaceholder = () => (
```

Comment:
```suggestion
const OfferThumbnailPlaceholder = () => (
```
Cover est le bon terme pour une jaquette de livre; ici, on pourrait utiliser Thumbnail, comme dans le retour d'`api`